packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -0,0 +1,447 @@
+#include "pch.h"
+#include "resource.h"
+
+#include <PathCch.h>
+#include <shlobj.h>
+#include <shobjidl.h>
+#include <windows.h>
+
+#include <ReactUWP/IXamlRootView.h>
+#include <ReactUWP/InstanceFactory.h>
+#include <ReactUWP/ReactUwp.h>
+#include <Unicode.h>
+
+#include <react-native-windows-extended.h>
+
+#pragma push_macro("GetCurrentTime")
+#undef GetCurrentTime
+
+#include <Windows.UI.Xaml.Hosting.DesktopWindowXamlSource.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Hosting.h>
+
+#pragma pop_macro("GetCurrentTime")
+
+namespace {
+
+struct LocalFreeDeleter {
+  void operator()(void *pv) const noexcept {
+    ::LocalFree(pv);
+  }
+};
+
+std::unique_ptr<WCHAR, LocalFreeDeleter> PathCombine(PCWSTR pathIn, PCWSTR more) {
```

Comment:
nit: why not use std::filesystem:: path APIs instead of Win32?

Reply:
Because I've never used `std::filesystem` and it didn't occur to me. 🤦‍♂️ I am so confused by these `PathCch.h` APIs... they were new in Win8 but they still used `::LocalFree` when they could've been sane and used the process heap. Or `CoTaskMem` since they return `HRESULT`s and look vaguely COM-y. Mysteries...